### PR TITLE
Feat: Changing control flow naming to tilde

### PR DESCRIPTION
**Pull request by 01Parzival10**
Changing control flow naming to ~ from empty.
Adds additional ~ for multiple incoming control flows to ensure unique names.

### CODE REVIEW

**Comment by 01Parzival10** [comment]
This fixes the converter side of DataFlowAnalysis/WebEditor#51.
The Editor still needs to be adjusted.

However, I suggest we merge this before the Editor is adjusted. While both the old and new syntax dont get accepted by the Editor, the new at least allows From-PCM-Created-Jsons to be converted into DFD's again

**Comment by sebinside** [comment]
Needs adjustments from @Kr0nox in the web editor

**Comment by Nicolas-Boltz** [comment]
@sebinside @Nicolas-Boltz @uuqjz review plz

File: bundles/org.dataflowanalysis.converter/src/org/dataflowanalysis/converter/DataFlowDiagramConverter.java [hunk 1]
```diff
@@ -263,22 +264,50 @@ private void createNodes(DataFlowDiagram dataFlowDiagram, List<Child> children,
...
     }
+    
+    private HashMap<Flow, String> createControlFlowNameMap(DataFlowDiagram dataFlowDiagram) {
+    	var controlFlowNameMap = new HashMap<Flow, String>();
+    	dataFlowDiagram.getNodes().forEach(node -> {
+    		var controlFlows = dataFlowDiagram.getFlows().stream()
+	    		.filter(flow -> flow.getDestinationNode().equals(node))
+	    		.filter(flow -> flow.getEntityName().equals("")).toList();
+    		
+    		String controlFlowName = "" + CONTROL_FLOW_NAME;
```
Comment (uuqjz) [inline bundles/org.dataflowanalysis.converter/src/org/dataflowanalysis/converter/DataFlowDiagramConverter.java]: Ommit the `"" +`

File: bundles/org.dataflowanalysis.converter/src/org/dataflowanalysis/converter/DataFlowDiagramConverter.java [hunk 2]
```diff
@@ -263,22 +264,50 @@ private void createNodes(DataFlowDiagram dataFlowDiagram, List<Child> children,
...
+    		for (var flow : controlFlows) {
+    			controlFlowNameMap.put(flow, controlFlowName);
+    			controlFlowName += CONTROL_FLOW_NAME;
```
Comment (uuqjz) [inline bundles/org.dataflowanalysis.converter/src/org/dataflowanalysis/converter/DataFlowDiagramConverter.java:284]: Why do you store "~", then "~~", "~~~" and so on. I do not understand the purpose.

**Review by uuqjz — CHANGES_REQUESTED** [review]
Found two things.

File: bundles/org.dataflowanalysis.converter/src/org/dataflowanalysis/converter/DataFlowDiagramConverter.java (hunk 1 above)
Comment (01Parzival10) [reply bundles/org.dataflowanalysis.converter/src/org/dataflowanalysis/converter/DataFlowDiagramConverter.java]: Done

**Review by uuqjz — APPROVED** [review]
LGTM

**Review by Nicolas-Boltz — APPROVED** [review]
looks good to me as well

**Pull request merged**